### DISCUSSION TITLE
Unable to log in to /admin

### DISCUSSION THREAD

**Discussion by mattyx0011** [ether/etherpad-lite#5903] [Q&A]
After installing Etherpad using Docker, I can use it normally, but I cannot log in to the web /admin."
I have already modified the contents of ' nano /opt/etherpad/etherpad-lite/settings.json' and removed the '/* */'
After making the modifications, restart Docker using 'docker restart etherpad.'
image: etherpad/etherpad:latest

The following is my content.

"users": {
"admin": {
// 1) "password" can be replaced with "hash" if you install ep_hash_auth
// 2) please note that if password is null, the user will not be created
"password": "admin",
"is_admin": true
},
"user": {
// 1) "password" can be replaced with "hash" if you install ep_hash_auth
// 2) please note that if password is null, the user will not be created
"password": "admin",
"is_admin": false
}
},

Please assist. Thank you.

**Comment by SamTV12345** [comment]
Hi,

as far as I can see is that you missspelled the word 'admin'. Can you login with 'aadmin'?

**Reply by mattyx0011** [reply]
Sorry for the error in the copying process, the settings.json file is correct "admin". https://imgur.com/a/2YIIW1f

**Reply by SamTV12345** [reply]
Ok. I was able to login succesfully with that user. Which database backend are you using?

**Reply by mattyx0011** [reply]
Sorry for the late response I was away for the weekend for three days. I'm not a professional Linux user so I followed the steps on this website to set it up step by step. https://www.digitalocean.com/community/tutorials/how-to-install-the-etherpad-collaborative-web-editor-on-ubuntu-20-04
Would you recommend me to start over, or is there a better approach? thank you.

**Reply by SamTV12345** [reply]
Hm. That tutorial doesn't seem to use Docker and rather a bare metal installation. According to your initial post I though you would use docker.

**Reply by mattyx0011** [reply]
YES! I've figured out my problem already! Sorry, I'm a beginner and it took me a few days to research.
Is it okay if I ask a question about a problem I'm having?After installing a Tables plugin, I'm encountering an error when trying to use the timeslider function. Should I open a new question to discuss this, or is it okay to continue here?Thank you.
https://imgur.com/a/Bwpq1PR

**Reply by SamTV12345** [reply]
Which plugin did you use? Occurred this because of the plugin or because of Etherpad?

**Reply by mattyx0011** [reply]
I've tried using various table plugins like ep_tables, ep_tables2 through ep_tables5, and the issue remains the same. Whenever I use a table plugin, I can't use the timeslider. However, when I remove the table plugin, it works fine

**Reply by SamTV12345** [reply]
Seems like to be an incompability. I couldn't find a version that is currently maintained. I don't have a computer within reach to test. I could give you feedback later this week. Otherwise you can try to contact one of the repository owners of the plugins.

**Reply by mattyx0011** [reply]
I think I have a rough idea of what the issue is, and I've managed to resolve it. Thank you for your assistance. I will close this discussion.
